utilities/unittests/test_pytest_utils.py

```
@@ -2816,6 +2818,64 @@ def test_no_deselection_when_no_multiarch_tests(self):
 
+class TestIsS390xOnly:
+    """Test cases for is_s390x_only function."""
+
+    def test_returns_true_for_cpu_arch_s390x_option(self):
+        config = MagicMock()
+        config.getoption.return_value = "s390x"
+
+        assert is_s390x_only(pytest_config=config) is True
+
+    @patch("utilities.pytest_utils.get_cluster_architecture", return_value={S390X})
+    def test_returns_true_for_s390x_cluster(self, _mock_get_cluster_architecture):
```

Ruff (0.16.0)

[warning] Fixture `_mock_get_cluster_architecture` without value is injected as parameter, use `@pytest.mark.usefixtures` instead

(PT019)